backend/src/main/kotlin/cz/loono/backend/db/model/SelfExaminationRecord.kt

```diff
@@ -0,0 +1,43 @@
+package cz.loono.backend.db.model
+
+import cz.loono.backend.api.dto.SelfExaminationResultDto
+import cz.loono.backend.api.dto.SelfExaminationStatusDto
+import cz.loono.backend.api.dto.SelfExaminationTypeDto
+import org.hibernate.envers.Audited
+import java.time.LocalDate
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "selfexamination_record")
+@Audited
+data class SelfExaminationRecord(
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    val id: Long = 0,
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    val type: SelfExaminationTypeDto = SelfExaminationTypeDto.BREAST,
+
+    @Column
+    val dueDate: LocalDate? = null,
```

Comment:
Je to dueDate ako "nasledujúci termín vyšetrenia"? Nebolo by dobré si zapísať aj prvý termín? Aby sme vedeli, odkedy počítať další mesiac.
Obecne mám pocit, že je tu málo políčok 😄 . Ale iné už neviem identifokovať :).

Reply:
Je jedno, jak tomu budes rikat - dueDate / plannedDate. staci jedno datum. prvni vysetreni ho urci a pri kazdy dalsi iteraci ho pouzijeme pro urceni datumu te dalsi. V tomhle PR chybi ta cast sbirani vysledku pri potvrzeni a posilani zpatky informace o prirazenych bodech a badges.